File: bus.py
def show_person(person):
    ticket = 12 * person
    print("*********************")
    print(f"Number of persons is {person}")
    print(f"Amount collected is {ticket} $")
    print("*********************")

def add_person():
    while True:
        try:
            n_person = int(input("Enter number of persons to be added: "))
            if n_person >= 0:
                return n_person
            else:
                print("*********************")
                print("Enter a positive number")
                print("*********************")
        except ValueError:
            print("*********************")
            print("Invalid input. Please enter a number.")
            print("*********************")

def remove_person(person):
    while True:
        try:
            n_person = int(input("Enter number of persons to be removed: "))
            if n_person < 0:
                print("*********************")
                print("Number of persons must be greater than or equal to 0")
                print("*********************")
            elif n_person > person:
                print("*********************")
                print("Number of persons to be removed should not be greater than current number of persons")
                print("*********************")
            else:
                return n_person
        except ValueError:
            print("*********************")
            print("Invalid input. Please enter a number.")
            print("*********************")

def main():
    person = 0
    is_running = True
    while is_running:
        print("*********************")
        print("   Bus Management    ")
        print("1. Show Person")
        print("2. Add Person")
        print("3. Remove Person")
        print("4. Exit")
        print("*********************")
        
        try:
            choice = int(input("Enter your choice (1-4): "))
            if choice == 1:
                show_person(person)
            elif choice == 2:
                person += add_person()
            elif choice == 3:
                person -= remove_person(person)
            elif choice == 4:
                is_running = False
            else:
                print("*********************")
                print("Enter a valid choice")
                print("*********************")
        except ValueError:
            print("*********************")
            print("Invalid input. Please enter a number.")
            print("*********************")
    
    print("*********************")
    print("Thank You! Have a nice day!")
    print("*********************")

if __name__ == "__main__":
    main()

File: bus.js
// Function to display the current number of persons and amount collected
function showPerson(person) {
    const ticket = 12 * person;
    console.log("*********************");
    console.log(`Number of persons is ${person}`);
    console.log(`Amount collected is ${ticket} $`);
    console.log("*********************");
}

// Function to add persons
function addPerson() {
    while (true) {
        const input = prompt("Enter number of persons to be added:");
        const nPerson = parseInt(input, 10);
        if (!isNaN(nPerson) && nPerson >= 0) {
            return nPerson;
        } else {
            console.log("*********************");
            console.log("Enter a positive number");
            console.log("*********************");
        }
    }
}

// Function to remove persons
function removePerson(person) {
    while (true) {
        const input = prompt("Enter number of persons to be removed:");
        const nPerson = parseInt(input, 10);
        if (isNaN(nPerson)) {
            console.log("*********************");
            console.log("Invalid input. Please enter a number.");
            console.log("*********************");
        } else if (nPerson < 0) {
            console.log("*********************");
            console.log("Number of persons must be greater than or equal to 0");
            console.log("*********************");
        } else if (nPerson > person) {
            console.log("*********************");
            console.log("Number of persons to be removed should not be greater than current number of persons");
            console.log("*********************");
        } else {
            return nPerson;
        }
    }
}

// Main function to run the bus management system
function main() {
    let person = 0;
    let isRunning = true;

    while (isRunning) {
        console.log("*********************");
        console.log("   Bus Management    ");
        console.log("1. Show Person");
        console.log("2. Add Person");
        console.log("3. Remove Person");
        console.log("4. Exit");
        console.log("*********************");

        const choice = parseInt(prompt("Enter your choice (1-4):"), 10);
        switch (choice) {
            case 1:
                showPerson(person);
                break;
            case 2:
                person += addPerson();
                break;
            case 3:
                person -= removePerson(person);
                break;
            case 4:
                isRunning = false;
                break;
            default:
                console.log("*********************");
                console.log("Enter a valid choice");
                console.log("*********************");
        }
    }

    console.log("*********************");
    console.log("Thank You! Have a nice day!");
    console.log("*********************");
}

// Run the main function
main();